### PR TITLE
begin menu harmonization

### PR DESCRIPTION
**Pull request by 0xF013**
[`* `#15711 `harmonize-action-and-dropdown-menus`](http://community.openproject.org/work_packages/15711)
- [x] remove 'withDropdown' directive
- [x] remove unused tests
- [x] fix specs
- [x] fix gear button menu
- [x] replace 'more' menu
- [x] replace tasks dropdown
- [x] replace query title menu
- [x] fix needing to press twice to tab through the menu items
- [x] fix no focus on opening menu if another instance is already visible
- [x] fix regression of right click menu positioning
- [x] push contextmenu plugin as well
- [ ] revert 0xf013 bower repository
- [x] remove commented lines
- [x] fix trap in non-context menu
- [x] add menu accessibility attributes
- [x] dropdown directive?
- [x] check scrollable dropdown 
- [x] remove git relic
- [x] fix save & save as
- [x] menu positions during scrolling
- [x] open only one menu
- [x] remove options-dropdown directive
- [x] rename test
- [ ] drop hasContextMenu or move hasDropdownDirective to /finnlabs/angular-context-menu/

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage. <a href='http://0xf013-feature-15711-harmonize-action-and-dropdown-menus.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Click to open.</a>

**Comment by 0xF013** [comment]
@myabc @ulferts I think this is ready for a review while I fix the specs

File: frontend/app/work_packages/controllers/menus/tasks-dropdown-menu-controller.js~8eb2cbde781862e4ae5a9c9a698528035d119a0a [hunk 1]
```diff
@@ -0,0 +1,33 @@
+//-- copyright
```
Comment (ulferts) [inline frontend/app/work_packages/controllers/menus/tasks-dropdown-menu-controller.js~8eb2cbde781862e4ae5a9c9a698528035d119a0a]: This seems to be some git relic.

**Comment by ulferts** [comment]
Looks pretty good functionality-wise (I am no judge when it comes to the code). The only two things I noticed:
- The context menu (right click inside wp table) is not robust with regards to scrolling
- It is possible to open all menus at the same time. When I open the menu for "+ Work package" and the table menu at the top right corner the menu opened first is not robust with regards to increasing/descreasing the window width. I believe the fix for that would be to only allow one window to be open at the same time.

**Comment by 0xF013** [comment]
@myabc @ulferts aside from minor discussion points it is ready to be reviewed

**Comment by myabc** [comment]
Looking :+1: functionality-wise.

**Comment by 0xF013** [comment]
god bless

**Pull request merged**